### DISCUSSION TITLE
Low version problems

### DISCUSSION THREAD

**Discussion by zhl-tcsec** [groupe-edf/nexus-report-plugin#31] [Q&A]
Whether the adapter Nexus OSS 3.15.0 Version ? What should I do if I want to fit myself ? I tried to modify the Maven dependency, but it didn't work. Please help me, thank you!

**Comment by mat1e** [comment]
Hi @zhl-tcsec, unfortunally I think it is not possible because somes features of the Nexus OSS used by this plugin appears only after the version 3.20.0 of Nexus OSS. 
You must update your installation to 3.24.0 minimum.

If you really want to fit it with your version, you can keep only the json api part of the plugin (the ui is dependent of nexus-ui-plugin and it is available only at 3.20.0 ...).

Maybe you can try to build a new project with the dependencies you need, and integrate the java code. The api part is in ReportResource.java and The service wich get all infos is ReportService.java.

I didn't find a way to adapt the plugin directly.

**Comment by zhl-tcsec** [comment]
Thanks, I don't need UI interface, I just need API, I will try it by myself, thanks again for your answer.
